src/fasta-recs.py

```diff
@@ -11,7 +12,50 @@ def main():
     )
     args = argparser.parse_args()
 
-    print(f"Now I need to process the records in {args.fasta}")
+    # Read the content
+    content = args.fasta.read()
+    
+    index = 0
+
+    while(index<len(content)):
+        # Look for '> *'
+        match = re.search(">(\s)*", content[index:])
+        if match is None:
+            break
+        index += match.end()
+
+        beginName = index
+
+        # Look for the end of the name
+        match = re.search("\n", content[index:])
+        if match is None:
+            chain = content[beginName:len(content)]
+            chain = re.sub('(\s)+',' ',chain).strip()
+            print(chain, end="\t")
+            break
+        index += match.start()
+
+        chain = content[beginName:index]
+        chain = re.sub('(\s)+',' ',chain).strip()
+        print(chain, end="\t")
+
+        beginChain = index        
+        # Look for the genome chain
+        match = re.search(">", content[index:])
+        #print(beginChain)
+        if match is None:
+            chain = content[beginChain:len(content)]
+            chain = re.sub('\n','', chain) # Delete intros
+            chain = re.sub('(\s)+',' ',chain).strip() # Replace spaces
+            print(chain)
+            break
```

Comment:
Except for explicitly removing newlines, this is duplicated code. You have the same code a little above. Generally, we prefer to reuse the same code rather than have it multiple places in the source code, especially if we need to fix it later... You could write a function that extracts the next sequence and reuse it.